core/trino-main/src/main/java/io/trino/sql/planner/iterative/rule/PushPredicateIntoTableScan.java

```diff
@@ -238,9 +238,7 @@ public static Optional<PlanNode> pushFilterIntoTableScan(
         }
 
         if (newDomain.isNone()) {
-            // TODO: DomainTranslator.fromPredicate can infer that the expression is "false" in some cases (TupleDomain.none()).
-            // This should move to another rule that simplifies the filter using that logic and then rely on RemoveTrivialFilters
-            // to turn the subtree into a Values node
```

Comment:
There is a similar comment in `RemoveRedundantPredicateAboveTableScan`. should it also be removed now?

Reply:
Missed that, removed

Reply:
Yes, I've missed that, thanks.